client/client-lib/Cargo.toml

```diff
@@ -31,6 +31,7 @@ fedimint-ln-client = { path = "../../modules/fedimint-ln-client" }
 fedimint-logging = { path = "../../fedimint-logging" }
 fedimint-mint-client = { path = "../../modules/fedimint-mint-client" }
 fedimint-wallet-client = { path = "../../modules/fedimint-wallet-client", default-features = false }
+fedimint-wallet-common = { path = "../../modules/fedimint-wallet-common", default-features = false }
```

Comment:
Not a blocker but my plan was for client and server always to re-export common, so downstream users should always import just client or server.